sda-commons-client-oidc/src/test/resources/fixtures/tokenResponse.json

```diff
@@ -0,0 +1,5 @@
+{
+  "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI1VFVDOVlZOXdXRk0xNzZqT2RvcWRYUWtUMmxNbklXYTh6MU5qVlVrNm5ZIn0.eyJleHAiOjE2MTI0MjkwMzksImlhdCI6MTYxMjQyODk3OSwianRpIjoiNWNkZmQ4YTAtYWUyYS00OGQxLWE3YmItZDgwY2I5NjRhODAxIiwiaXNzIjoiaHR0cHM6Ly9rZXljbG9hay5zZGFkZXYuc2RhLXNlLmlvL2F1dGgvcmVhbG1zL21hbHdhcmUtc2Nhbm5lciIsImF1ZCI6Im1hbHdhcmUtc2Nhbm5lciIsInN1YiI6IjJiZjY2MzQzLWUyMDQtNDc1NS1iZTFkLWU3OWVjZTAyZTQ4NCIsInR5cCI6IkJlYXJlciIsImF6cCI6Im1hbHdhcmUtc2Nhbm5lciIsInNlc3Npb25fc3RhdGUiOiJkMGZjYjgxNi02MjU3LTQ4NzEtYTg1MC1hYjI2MjFhNjg1ZWYiLCJhY3IiOiIxIiwiYWxsb3dlZC1vcmlnaW5zIjpbIioiXSwic2NvcGUiOiJlbWFpbCBwcm9maWxlIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInByZWZlcnJlZF91c2VybmFtZSI6Im1hbHdhcmUtc2Nhbm5lci1lbmQyZW5kIn0.yMVQCgZkDA7-glUaJVNrAt65eei3r7_OOVX2DCXcA33dVw7XXy5a3WkY-IsbJiaH1euFU8-Vw_a5JCP2QB94AJxWpa5fSMKytspXQKeKf6FtiOHRtvRfpNMvwv2zuEzITJg-IqFg3yo4Bpfhmepv1lvxdPu6ip1u_OznRX_-uR829mfq-J7SW81RSTaxDmDVweoIDSi24tdDkJqjOET3PziVWhhY8AqJUmUFf41mHhM2P5w-52KZldSbvuPN32Z5pb-MJv_GonNei2B-8IbWdHml_v7HVlB4JYCOCyABWIYrorKYnxDeFc8j4NungzMQJ9_uwRcN1jatVqKBjt0l1w",
+  "expires_in": 60,
```

Comment:
Ah, I see. It's from the OAuth spec:
https://tools.ietf.org/html/rfc6749#page-73